nio_cli/commands/blockcheck.py

```diff
@@ -0,0 +1,241 @@
+import json
+import os
+import sys
+import subprocess
+
+from .base import Base
+
+
+class BlockCheck(Base):
+
+    # This should be run from inside the block dir
+
+    def __init__(self, options, *args, **kwargs):
+
+        super().__init__(options, *args, **kwargs)
+        self._block = os.getcwd().split('/')[-1]
```

Comment:
This isn't necessarily the `block`. Maybe change this variable name to something `directory_name` or something.